src/components/LoginForm/index.js

```diff
@@ -95,22 +115,33 @@ function Login({ providers, ...props }) {
                     })
                   }
                 >
+                  <Image height={45} width={45} src={GoogleIcon} alt="" />
                   Sign in with {provider.name}
                 </Button>
               ))}
           </div>
-        </form>
+          <Typography className={classes.text}>{password}</Typography>
```

Comment:
```suggestion
          <Typography className={classes.text}>{forgotPasswordPrompt}</Typography>
```
This should also be a link or button i.e. active.